wizard-js/source-parser.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const deepEqual = require('deep-equal');
+const uniq = require('uniq');
+const { flatten } = require('array-flatten');
```

Comment:
can we get away using the spread operator instead of an extra dependency?

Reply:
I _believe_ the spread operator doesn't let you flatten an individual array in a variable without additional wrapping code (e.g. a `reduce()` operation).